frontend/app/components/encryptable/form.js

```diff
@@ -151,4 +160,8 @@ export default class Form extends BaseFormComponent {
       AccountValidations
     );
   }
+
+  get invertedWithSymbols() {
```

Comment:
Is this really needed or can you just do it at another point?
If yes why not name it `notWithSymbols`